packages/ember-mirage/addon/start-mirage.js

```diff
@@ -2,7 +2,7 @@ import { pluralize, singularize } from 'ember-inflector';
 
 import assert from './assert';
 
-export default function startMirage(makeServer, { owner, env } = {}) {
+export default function startMirage(makeServer, { owner, env, ...otherOptions } = {}) {
```

Comment:
any additional options that the user might want to pass on were not being passed on.